internal/overlord/servstate/manager_test.go

```diff
@@ -765,7 +765,7 @@ services:
 
 	// Start service and wait till it starts up the first time.
 	chg := s.startServices(c, []string{"test2"}, 1)
-	svc := s.waitUntilService(c, "test2", func(svc *servstate.ServiceInfo) bool {
+	s.waitUntilService(c, "test2", func(svc *servstate.ServiceInfo) bool {
```

Comment:
We weren't actually using the return value from this helper function, so I just got rid of it (it was always non-nil anyway, otherwise it fatal'd the test).